tests/test_fmi3.py

```diff
@@ -581,6 +580,36 @@ def test_get_input_list(self):
 
         assert expected == list(inputs.keys())
 
+    def test_get_int32(self):
+        """ Test get int32. """
+        fmu_path = FMI3_REF_FMU_PATH / "Feedthrough.fmu"
+        fmu = load_fmu(fmu_path)
+        int32_vals = fmu.get_int32([19, 20])
+
+        np.testing.assert_equal(np.array([0, 0], dtype=np.int32), int32_vals)
```

Comment:
Consideration for these tests; should be also implement the setters and test with proper values. E.g., for the int64 test actually test a value that would not fit an int32?